src/sentry/api/endpoints/team_all_unresolved_issues.py

```diff
@@ -50,7 +57,7 @@ def calculate_unresolved_counts(team, project_list, start, end):
     # Grab the historical data bucketed by day
     new_issues = (
         Group.objects.filter_to_team(team)
-        .filter(first_seen__gte=start, first_seen__lt=end)
+        .filter(first_seen__gte=start, first_seen__lt=end, **group_enviornment_filter)
```

Comment:
If we have an environment, do we want to filter to the date the group was first seen in that environment?

Reply:
Actually... it probably doesn't make sense to do this, since we don't have other information in `GroupHistory` filtered by environment